storage/range_gc_queue.go

```diff
@@ -33,6 +33,11 @@ const (
 
 	// rangeGCQueueTimerDuration is the duration between GCs of queued ranges.
 	rangeGCQueueTimerDuration = 10 * time.Second
+
+	// RangeGCQueueUnleasedDuration is the default unlease duration threshold
+	// used for queuing decision. A range whose lease has been expired for
+	// longer than this duration will be queued. Exposed for testing.
+	RangeGCQueueUnleasedDuration = 1 * time.Hour
```

Comment:
Yes, the tests are in the `storage` directory, but the package is `storage_test`..